backend/src/interfaces/Ientities/Iusers.ts

```diff
@@ -14,6 +14,7 @@ interface IUser {
   courses: ObjectId[];
   group: string;
   employees: ObjectId[];
+  pendingCourses: ObjectId[];
```

Comment:
don't foget that sometimes it's field can containe populated objects, so you need add "or" and use type from prev